### PR TITLE
Docs readme

### PR DESCRIPTION
**Pull request by hsnice16**
This PR will add the updated README in the development branch.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # PoshUI Components
 
-### This repo contains the implementation of components of the [Posh UI](https://github.com/hsnice16/PoshUI-Documentation) component library.
+### This repo contains the implementation of components of the <a href="https://github.com/hsnice16/PoshUI-Documentation" target="_blank">Posh UI</a> component library.
```
Comment (sainath7878) [inline README.md]: I guess the former line is the right way to add links in markdown. It's fine if the latter one is working.

Comment (hsnice16) [reply README.md]: Thanks for pointing this, Sainath. I just got to know that `target="_blank"` does not work in github markdown. So I will change that.

**Review by anik31 — APPROVED** [review]
Everything looks good. You can merge it.

Suggestion - add links for individual components

**Comment by hsnice16** [comment]
> Everything looks good. You can merge it.
> 
> Suggestion - add links for individual components

Yeah, I have given in the documentation readme. Thanks for suggesting.

**Comment by hsnice16** [comment]
Thank you for reviewing this.

**Pull request merged**